<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>incomesChart</class>
 <widget class="QWidget" name="incomesChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Open Sans Condensed SemiBold</family>
    <pointsize>14</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>incomesChart</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(233, 234, 255)</string>
  </property>
  <widget class="QChartView" name="incomesGraphicsView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>1200</width>
     <height>750</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="chartIncomesLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>35</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Показать доходы с</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="chartFirstDateEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>35</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
    border-radius: 5px;
	border: 1px solid black;
}</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="chartIncomesLabel2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>35</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>по</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="chartSecondDateEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>35</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
    border-radius: 5px;
	border: 1px solid black;
}</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
